rtlib/src/Data.cpp

```diff
@@ -0,0 +1,74 @@
+//
+// Created by Anthony Tamasi on 2019-10-31.
+//
+
+#include <itkOpenCVImageBridge.h>
+
+#include "rt/Data.hpp"
+#include "rt/io/OBJReader.hpp"
+
+
+using namespace Data;
+
+// IO
+using OCVBridge = itk::OpenCVImageBridge;
+
+
+Data::Pointer Data::Load(const fs::path& path)
+{
+    //HOW TO GET METADATA AKA FILE TYPE? Could just read the 'metadata.json' file from the current directory right? Since each
+    //input data thing in the RegPkg is a sub-folder that contains only the data and then the metadata file.
+    //Read in metadata file
+
+    //Select type of object based on this file's type
+    switch(metadata.type){
+        case "image":
+            return new Image(path); //SHOULD THESE BE SHARED POINTERS INSTEAD OF "new"? e.g. return std::make_unique<Image>(path);?
+        case "envi":
+            return new ENVI(path);
+        case "mesh":
+            return new Mesh(path);
+    }
+}
+
+
+Data::Image(const fs::path& path)
+{
+    img_ = cv::imread(path.string());
+}
+
+
+Data::ENVI(const fs::path& path)
+{
+    //Read in ENVI file from the file path, set the object to the envi_ private member
+    envi_ = et::ENVI envi(path); //or does it need to be envitools::ENVI envi(path);
```

Comment:
`envitools` is the namespace. If you want to use et, then add:
```c++
namespace et = envitools
```
at the top of the file.

Also, you can probably use initializers before the function:
```c++
Data::ENVI(const fs::path& path) : envi_{path} {}
```